src/style/components/list/conversation-list-calling-cell.less

```diff
@@ -154,16 +154,20 @@
       }
 
       .verified-icon,
-      .guest-icon {
+      .guest-icon svg {
+        display: flex;
```

Comment:
Don't know the context, but can't it be just `block`?

Reply:
Align-item is used on the parent element, it's subtle but it gets off center with block